src/exprestive.coffee

```diff
@@ -89,7 +89,7 @@ class Exprestive
 
   # Save a function to @reversePaths to get a url back from a route name
   registerReverseRoute: ({routeName, url}) ->
-    @reversePaths[camelCase routeName] = (id) ->
+    @reversePaths[routeName] = (id) ->
```

Comment:
What do you think about doing something like this
```coffeescript
for name in [routeName, camelCase(routeName), snakeCase(routeName)]
  @reversePaths[name] = (id) ->
```

This way the restful routes can be called with either camel or snake case?